clients/cloud/groovy/src/main/groovy/io/confluent/examples/clients/cloud/ProducerExample.groovy

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.confluent.examples.clients.cloud
+
+import groovy.transform.CompileStatic
+import io.confluent.examples.clients.cloud.model.DataRecord
+import io.confluent.kafka.serializers.KafkaJsonSerializer
+import org.apache.kafka.clients.admin.AdminClient
+import org.apache.kafka.clients.admin.NewTopic
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.apache.kafka.clients.producer.RecordMetadata
+import org.apache.kafka.common.errors.TopicExistsException
+import org.apache.kafka.common.serialization.StringSerializer
+
+import java.util.concurrent.ExecutionException
+
+import static io.confluent.examples.clients.cloud.util.PropertiesLoader.loadConfig
+import static java.lang.System.exit
+import static org.apache.kafka.clients.admin.AdminClient.*
+import static org.apache.kafka.clients.producer.ProducerConfig.*
+
+@CompileStatic
+class ProducerExample {
+
+  // Create topic in Confluent Cloud
+  static void createTopic(String topic,
+                          int partitions,
+                          int replication,
+                          Properties cloudConfig) {
+
+    def newTopic = new NewTopic(topic, partitions, (short) replication)
+
+    def adminClient = create cloudConfig
+
+    adminClient.withCloseable {
+      it.createTopics([newTopic]).all().get()
```

Comment:
`adminClient` rather than `it` would be better style?

there's some logic related to the topic already existing in the java example that's not here. i guess you skipped that on purpose? might want to put that in (but I'm not going to block the pr on it).

Reply:
oh, i see you do that later down.
very good. I think where you put this is actually more appropriate than the java example.